project_config/urls.py

```diff
@@ -13,8 +13,10 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
-from django.urls import path
+from django.urls import path,include
 
-urlpatterns = [
 
+urlpatterns = [
+    path('', include('list_project.urls')),
```

Comment:
update to `path('list_project', include('list_project.urls'))`,

Reply:
done